### ISSUE TITLE
Installation Manager

### ISSUE THREAD

**Issue by Lefuneste83** [platformio/platformio-core-installer#138]
%23 Description of problem
Unable to update to V2.X in VS Code

  BEFORE SUBMITTING, PLEASE SEARCH FOR DUPLICATES IN
  - https://github.com/platformio/platformio-vscode-ide/issues%3Fq=is%3Aissue 

  %23 Configuration

  VSCode: 1.49.1
  PIO IDE: v2.1.0
  System: Linux, 5.4.0-48-generic, x64

  %23 Exception
  ```
  Error: Error: Compatible PlatformIO Core not found.
Reason: Could not import PlatformIO module. Error: Traceback (most recent call last):
  File "<string>", line 3, in <module>
ModuleNotFoundError: No module named 'platformio'


	at /home/user/.vscode/extensions/platformio.platformio-ide-2.1.0/node_modules/platformio-node-helpers/dist/index.js:1:4911
	at ChildProcess.s (/home/user/.vscode/extensions/platformio.platformio-ide-2.1.0/node_modules/platformio-node-helpers/dist/index.js:1:4807)
	at ChildProcess.emit (events.js:223:5)
	at maybeClose (internal/child_process.js:1021:16)
	at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
  ```

**Comment by Lefuneste83** [comment]
python3 --version
Python 3.6.9

**Comment by Lefuneste83** [comment]
Solved after upgrading python3 ro 3.8
